client/my-sites/domains/domain-management/components/domain/domain-info-card/transfer/index.tsx

```diff
@@ -15,7 +14,7 @@ const DomainTransferInfoCard = ( {
 
 	if (
 		! domain.currentUserIsOwner ||
-		( domain.expired && ! isDomainInGracePeriod( domain ) ) ||
+		( ! domain.isRenewable && ! domain.isRedeemable ) ||
```

Comment:
`is_redeemable` is true only during the exact redemption period, so we have to check for `is_renewable` too.